File: baekjoon2577.py
import sys

num1 = int(sys.stdin.readline())
num2 = int(sys.stdin.readline())
num3 = int(sys.stdin.readline())

total = num1 * num2 * num3
numbers = str(total)

res = {}

for i in range(10):
    n = str(i)
    res[i] = 0
    for j in range(len(numbers)):
        if n == numbers[j]:
            res[i] += 1

for v in res.values():
    print(v)

File: baekjoon2577.c
#include <stdio.h>

void main()
{
    int num1, num2, num3;
    scanf("%d %d %d", &num1, &num2, &num3);
    int sum = num1 * num2 * num3;
    char numbers;
    // numbers = (char)sum;
    printf("%d\n%d\n%d\n", num1, num2, num3);
    printf("num : %d\n", sum);
    // printf("%c\n", numbers);
}